src/MusicStore/Areas/Admin/Controllers/StoreManagerController.cs

```diff
@@ -8,8 +8,10 @@
 using Microsoft.AspNetCore.Mvc.Rendering;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
 using MusicStore.Models;
 using MusicStore.ViewModels;
+using MusicStore.Extensions;
```

Comment:
Sort & Remove namespaces in all updated files. (But keep System.* namespaces on top.)